xrm/xrm.d.ts

```diff
@@ -912,12 +882,9 @@ declare namespace Xrm
             text: string;
 
             /**
-             * The value, as a string.
-             *
-             * @remarks     You must use parseInt to convert this value to a number before you can use it to
-             *              set the value of an OptionSetAttribute.
+             * The value, as a number
              */
-            value: string;
+            value: number;
```

Comment:
It's a Turbo Forms related change:  Please see @6ix4our and my discussion: https://github.com/6ix4our/DefinitelyTyped/issues/3.

By default now in new versions of CRM 2015 U1 and CRM 2016, turbo forms is enabled resulting a number type, and switching it to legacy, results in a string type.

I guess it could be `string|number` until the legacy forms is removed entirely, but number is the default behavior in 7.1 and above.